site/learn/expert-advice/authentication/webauthn-explained.md

```diff
@@ -0,0 +1,515 @@
+---
+layout: advice
+title: WebAuthn Explained
+description: An in-depth look at how WebAuthn can help you add biometric authentication to your website.
+author: Dan Moore
+image: advice/vendor-lockin/expert-advice-avoiding-authentication-system-lock-in-header-image.png
+category: Authentication
+date: 2022-09-28
+dateModified: 2022-09-28
+---
+
+WebAuthn is a new way to authenticate your users using biometric, secure authentication methods. WebAuthn requires something called an authenticator. What is an authenticator? An authenticator is a hardware or software device that is used to authenticate a user, via biometric or other methods, and that follows certain standards.
+
+That's a bit tautological, isn't it? Well, yes, but read on for more details. Also, clowns.
+
+## WebAuthn Overview
+
+Let's start off with an overview of the use cases, "ceremonies", and types of authenticators that all play a role in a WebAuthn implementation.
+
+There are three main use cases that WebAuthn works well for.
+
+1. Re-authentication, where the user has an account created in another way, but wants an easier way to login.
+1. First factor authentication, where the user has no account, but uses WebAuthn to login instead of a more typical first factor like a username/password combination.
```

Comment:
First factor or "bootstrap" authentication also applies when the user has an account and authenticates on a new device for the first time.

A user-verifying authenticator can provide two factors of authentication in a single step, so you can jump right in rather than having to provide a password plus TOTP (or similar).